README.md

```diff
@@ -322,7 +345,7 @@ This method takes the following request:
     const filtersByParty = {};
     filtersByParty[sender] = { inclusive: { templateIds: [PING, PONG] } };
     const request = {
-        begin: { boundary: da.LedgerOffset.Boundary.END },
+        begin: { boundary: ledger.LedgerOffsetBoundaryValue.BEGIN },
```

Comment:
Is the switch from `END` to `BEGIN` on purpose?

Reply:
Yeah, I'll clarify as this is why I had to come up with the complex explanation afterwards, thanks.

Reply:
Hopefully better explained in https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/31c20f8b85cf929172f994519a377111f011865f

Reply:
Improved on with https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/7cbd34f6f64471bf6912614e32c97740d4aff4ac